orangecontrib/timeseries/tests/test_timeslice.py

```diff
@@ -0,0 +1,13 @@
+from Orange.widgets.tests.base import WidgetTest
+
+from orangecontrib.timeseries import Timeseries
+from orangecontrib.timeseries.widgets.owtimeslice import OWTimeSlice
+
+
+class TestTimeSlice(WidgetTest):
+    def setUp(self):
+        self.widget = self.create_widget(OWTimeSlice)
+
+    def test_numeric_timedelta(self):
+        data = Timeseries.from_file('numericdt')
```

Comment:
This is missing the proper path prefix, hence the file is not found.

Reply:
`'../widgets/tests/datasets/numericdt.csv'`

Reply:
Fixed, this file shouldn't have been included.